package.xml

```diff
@@ -30,31 +30,34 @@
   <author>Aaron Blasdel</author>
   <author>Isaac Saito</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
   <exec_depend>rqt_action</exec_depend>
-  <exec_depend>rqt_bag</exec_depend>
-  <exec_depend>rqt_bag_plugins</exec_depend>
   <exec_depend>rqt_console</exec_depend>
-  <exec_depend>rqt_dep</exec_depend>
   <exec_depend>rqt_graph</exec_depend>
   <exec_depend>rqt_image_view</exec_depend>
-  <exec_depend>rqt_launch</exec_depend>
-  <exec_depend>rqt_logger_level</exec_depend>
   <exec_depend>rqt_msg</exec_depend>
   <exec_depend>rqt_plot</exec_depend>
   <exec_depend>rqt_publisher</exec_depend>
-  <exec_depend>rqt_py_common</exec_depend>
   <exec_depend>rqt_py_console</exec_depend>
   <exec_depend>rqt_reconfigure</exec_depend>
   <exec_depend>rqt_service_caller</exec_depend>
   <exec_depend>rqt_shell</exec_depend>
   <exec_depend>rqt_srv</exec_depend>
   <exec_depend>rqt_top</exec_depend>
   <exec_depend>rqt_topic</exec_depend>
+
+  <!-- Not yet ported to ROS 2
+  <exec_depend>rqt_bag</exec_depend>
+  <exec_depend>rqt_bag_plugins</exec_depend>
+  <exec_depend>rqt_dep</exec_depend>
+  <exec_depend>rqt_launch</exec_depend>
+  <exec_depend>rqt_logger_level</exec_depend>
+  <exec_depend>rqt_py_common</exec_depend>
   <exec_depend>rqt_web</exec_depend>
+  -->
```

Comment:
Instead of this separate block please comment out the existing lines if there is no ROS 2 port yet.